### DISCUSSION TITLE
HW 1 Part 1 Question 2

### DISCUSSION THREAD

**Discussion by chrisbergen4** [cooperrc/computational-mechanics#83] [General]
How are we supposed to format the answers? By purely copy and pasting, such as "str1==str2", it gives an output of "False", but should we do "print("B. False"----for example.

Thanks!

**Comment by cooperrc** [comment]
Formatting is up to you, but make sure you're output is viewable, e.g. 

```python
#---------------a cell-----------------------
str1 < str2
str1 == str2
#---------------------------------------------
```

will only print the 

```python 
Out [1]: False
```

Because it prints the last output, you'll need separate cells for each command. 

That's why we used `print` statements in some of the examples so we can choose what output is printed and add some description of what its supposed to be.
